projects/packages/forms/src/contact-form/class-editor-view.php

```diff
@@ -59,10 +60,8 @@ public static function grunion_media_button() {
 	 * @return array
 	 */
 	public static function mce_external_plugins( $plugin_array ) {
-		$plugin_array['grunion_form'] = Assets::get_file_url_for_environment(
-			'jetpack_vendor/automattic/jetpack-forms/dist/contact-form/js/tinymce-plugin-form-button.js',
-			'jetpack_vendor/automattic/jetpack-forms/dist/contact-form/js/tinymce-plugin-form-button.js'
-		);
+		$plugin_array['grunion_form'] = plugins_url( 'jetpack_vendor/automattic/jetpack-forms/dist/contact-form/js/tinymce-plugin-form-button.js', Jetpack_Constants::get_constant( 'JETPACK__PLUGIN_FILE' ) );
```

Comment:
How come we are not doing something like: `plugins_url( '../../dist/contact-form/js/tinymce-plugin-form-button.js', __FILE__ );` so we don't have to rely on the `JETPACK__PLUGIN_FILE` constant defined only by the Jetpack plugin?